app/app.js

```diff
@@ -1,17 +1,23 @@
 const express = require('express')
-const {getTopics} = require('./controller.js')
+const {getTopics, getArticles} = require('./controller.js')
 const app = express()
 app.use(express.json())
-app.use((error, request, response, next) =>
+app.use((err, request, response, next) => 
 {
-    if(error)
+    console.log(err)
+    response.status(err.status).send(err.msg)
+})
```

Comment:
This doesnt seem to be doing anything - it doesn't have any conditional logic and thus will hit on any request